docs/design/windows-design.md

```diff
@@ -188,18 +188,6 @@ these scenarios:
 * OpenFlow connection
 * The connection between CNI plugin and CNI server
 
-## Antrea Management on Windows
-
-### Antrea Agent Management
-
-The Antrea Agent is running as a process on the Windows Node, but it is managed using a DaemonSet. The utility
-[Rancher Wins](https://github.com/rancher/wins) is used to manage the host process from inside the DaemonSet Pod.
-The Antrea Agent is configured using a ConfigMap, and the environment variables are set by kubelet on Windows.
-
-### OVS Management
-
-OVS is running as 2 Windows Services: one for ovsdb-server and one for ovs-vswitchd.
```

Comment:
Instead of this line, we can add a liner that "ovs userspace processes i.e ovsdb-server and ovs-vswitchd run inside container wrt 1.13 and from 1.14 windows will support pristine host wrt windows ovs driver installation" , Design doc needs to be updated wrt ovs containerisation and pristine host support.

Reply:
I added back a paragraph.
"pristine host" is not a very helpful term IMO, we should avoid it and be more accurate

The doc definitely needs more updating, there is stale information, in particular when it comes to kube-proxy.